package.json

```diff
@@ -23,7 +23,9 @@
     "format": "$(npm bin)/prettier --write './**/*.js'",
     "pretest": "npm run format && npm run lint",
     "lint": "$(npm bin)/eslint .",
-    "test": "$(npm bin)/jest --coverage."
+    "test": "$(npm bin)/jest --coverage.",
```

Comment:
Fun new web-ui build targets

Reply:
Shouldn't that be `--coverage .` with a space? I'll have to try locally.